kernel/kernel-websubclient-api/src/main/java/io/mosip/kernel/websub/api/config/IntentVerificationConfig.java

```diff
@@ -40,6 +40,10 @@ public void setEmbeddedValueResolver(StringValueResolver resolver) {
 	public void setApplicationContext(ApplicationContext applicationContext) throws BeansException {
 		mappings = new HashMap<>();
 		for (String beanName : applicationContext.getBeanDefinitionNames()) {
+			//Skip processing this intentVerificationConfig bean.
+			if(beanName.equals("intentVerificationConfig")) {
+				continue;
+			}
```

Comment:
why are we skipping this ?

Reply:
This was causing circular reference issue after the migration. So this condition is added.

Reply:
But is the right way to handle the circular dependency ?

Reply:
Yes Vishwa. Here it was re-iterating the same bean again while processing other beans. So we skipped it.